samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -207,6 +208,13 @@ static void state_binding(u8_t lightness, u8_t temperature)
 	light_ctl_srv_user_data.lightness =
 		light_lightness_srv_user_data.actual;
 
+	if (light_lightness_srv_user_data.actual == 0) {
+		gen_onoff_srv_root_user_data.onoff = 0;
+	} else {
+		gen_onoff_srv_root_user_data.onoff = 1;
+	}
+
+jump:
```

Comment:
Could you rename this to something more descriptive. Would `update_temp` be accurate?

Reply:
Yes, I did it using "git rebase -i" commands :)